integration-tests/src/test/java/uk/gov/di/authentication/shared/services/AuthOrchSerializationServicesIntegrationTest.java

```diff
@@ -1,37 +1,93 @@
-    private final uk.gov.di.orchestration.shared.serialization.Json OrchobjectMapper =
-            uk.gov.di.orchestration.shared.services.SerializationService.getInstance();
+
+    private static final String REDIS_HOST =
+            System.getenv().getOrDefault("REDIS_HOST", "localhost");
+    private static final Optional<String> REDIS_PASSWORD =
+            Optional.ofNullable(System.getenv("REDIS_PASSWORD"));
+    private static final String BROWSER_SESSION_ID = "browser-session-id";
+    private static final String CLIENT_SESSION_ID = "client-session-id";
+
+    private uk.gov.di.orchestration.shared.services.SessionService orchSessionService;
+    private uk.gov.di.authentication.shared.services.SessionService authSessionService;
+
+    @BeforeEach
+    void setup() {
+        var orchConfigurationService =
+                uk.gov.di.orchestration.shared.services.ConfigurationService.getInstance();
+        var authConfigurationService =
+                uk.gov.di.authentication.shared.services.ConfigurationService.getInstance();
+        var orchRedisConnectionService =
+                new uk.gov.di.orchestration.shared.services.RedisConnectionService(
+                        REDIS_HOST, 6379, false, REDIS_PASSWORD, false);
+        var authRedisConnectionService =
+                new uk.gov.di.authentication.shared.services.RedisConnectionService(
+                        REDIS_HOST, 6379, false, REDIS_PASSWORD, false);
+
+        orchSessionService =
+                new uk.gov.di.orchestration.shared.services.SessionService(
+                        orchConfigurationService, orchRedisConnectionService);
+        authSessionService =
+                new uk.gov.di.authentication.shared.services.SessionService(
+                        authConfigurationService, authRedisConnectionService);
+    }
 
     @Test
-    void sessionsSerializedByAuthCanBeDeserializedByOrch()
-            throws Json.JsonException,
-                    uk.gov.di.orchestration.shared.serialization.Json.JsonException {
-        var authSession =
-                new Session("session-id")
-                        .withBrowserSessionId("browser-session-id")
-                        .addClientSession("client-session-id");
-        var orchSession =
-                new uk.gov.di.orchestration.shared.entity.Session("session-id")
-                        .withBrowserSessionId("browser-session-id")
-                        .addClientSession("client-session-id");
-
-        var authSerializedSession = authObjectMapper.writeValueAsString(authSession);
-        var orchDeserialized =
-                OrchobjectMapper.readValue(
-                        authSerializedSession, uk.gov.di.orchestration.shared.entity.Session.class);
-
-        Assertions.assertTrue(new ReflectionEquals(orchSession).matches(orchDeserialized));
+    void authAndOrchSessionsShouldBeAbleToHaveDifferentFieldsWithoutOverwritingOneAnother() {
```

Comment:
I think these can be separated into multiple tests, so it's more readable and we won't need code comments. Eg 
1) create orch session, assert auth can read it
2) visa versa
3) create orch session, auth reads and edits some shared property (if there are any), assert orch can still read it and has the shared property updated
4) visa versa
5) orch creates session with browserSessionId (or any other property only orch has), auth reads and stores the session, assert orch still has the browserSessionId (and perhaps that the auth session doesn't)
6) visa versa
7) create orch session, auth can read it, then orch cycles sessionId, and auth can still read it with the new sessionId

What do you reckon?

Reply:
Not a bad idea.

Reply:
Done